tachyon/zk/plonk/vanishing/graph_evaluator.h

```diff
@@ -207,7 +207,7 @@ class GraphEvaluator : public tachyon::zk::Evaluator<F, ValueSource> {
   }
 
   std::vector<F> CreateInitialIntermediates() const {
-    return std::vector<F>(num_intermediates_);
+    return std::vector<F>(num_intermediates_, F::Zero());
```

Comment:
Do we need to initialize this? It seems like `GraphEvaluator<F>::Evaluate()` fills the values.